### PR TITLE
Graphics and UI overhaul

### PR DESCRIPTION
**Pull request by vampcat**
Building upon Adri's work, this PR makes the ui and content drawn on screen resolution-agnostic.

### CODE REVIEW

**Comment by GooeyHub** [comment]
Uh oh, something went wrong with the build. Need to check on that

File: engine/src/main/java/org/destinationsol/common/Norm.java [hunk 1]
```diff
@@ -1,31 +1,28 @@
 /*
- * Copyright 2018 MovingBlocks
+ * Copyright 2017 MovingBlocks
```
Comment (Cervator) [inline engine/src/main/java/org/destinationsol/common/Norm.java]: Ack! Time stealing ninja!

**Comment by Cervator** [comment]
Huh, I started reviewing this but I think something something went funny. I'm seeing recently added changes getting removed in this PR, like the whole `Console` class and some initial steps toward making updates more entity-systemy? Or is that on purpose? Might need a sanity check before digging deeper.

**Comment by Adrijaned** [comment]
This branch was made based on some older branch, then because of git
history going funny, only the source code was copied to a branch based on a
newer commit. No deeper logic behind that


S pozdravem a přáním pěkného dne

Jindřich Dítě

2018-09-18 5:17 GMT+02:00 Rasmus Praestholm <notifications@github.com>:

**Comment by Cervator** [comment]
Closing as merged by redo PRs :-)

**Pull request closed without merging**